tests/release/pipelines/fbc_release.go

```diff
@@ -181,16 +182,20 @@ func assertReleasePipelineRunSucceeded(devFw, managedFw framework.Framework, dev
 	Eventually(func() error {
 		snapshot, err := devFw.AsKubeDeveloper.IntegrationController.GetSnapshot("", buildPr.Name, "", devNamespace)
 		if err != nil {
-			return fmt.Errorf("snapshot %s in namespace %s has not been found yet", snapshot.Name, devNamespace)
```

Comment:
Why wouldn't we want to print out the snapshot name anymore?

Reply:
The snapshot can't be got sometimes and using "snapshot.name" causes memory issue in this situation.

Reply:
Sometimes, the snapshot CR can't be got correctly.  It will report memory issue by using "snapshot.name" in that situation.